zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
Those are both extremely appreciated tips, thank you.

On Tue, Nov 3, 2015 at 11:58 AM, Richard Frank <notifications@github.com>
wrote:

> In zipline/algorithm.py
> <https://github.com/quantopian/zipline/pull/806#discussion_r43782433>:
>
> > @@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
> >          self.algoscript = kwargs.pop('script', None)
> >
> >          self._initialize = None
> > +        self._analyze = kwargs.pop('analyze', None)
>
> I don't know of a way to call Travis without requesting a pull, but if you
> don't want to open a PR against quantopian/zipline master, you could move
> your changes to a branch and PR against your own master.
>
> Running your branch locally, the issue I'm getting is about metadata
> missing from the assets db. I think if you pass in a panel directly to run,
> then it will update the db and create the data source for you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/zipline/pull/806/files#r43782433>.
>